app/controllers/groups_controller.rb

```diff
@@ -22,6 +22,15 @@ def update
     respond_with @group, location: -> { groups_path }
   end
 
+  def delete
```

Comment:
Do you need delete and destroy?

Reply:
Yep, just `destroy`

Reply:
I think it is meant to be `show`

Reply:
Yep, thanks, it was useless. I deleted it!